saleor/graphql/translations/types.py

```diff
@@ -146,13 +151,20 @@ def resolve_product_variant(root: product_models.ProductVariant, info):
 
     @staticmethod
     def resolve_attribute_values(root: product_models.ProductVariant, _info):
-        translatable_values = []
-        for assignment in root.attributes.all():
-            attr = assignment.attribute
-            if attr.input_type in AttributeInputType.TRANSLATABLE_ATTRIBUTES:
-                value = assignment.values.first()
-                translatable_values.append(value)
-        return translatable_values
+        def func(attributes):
```

Comment:
`func` is a good name when you're building a draft of some changes; here it should be renamed to something more understandable by others. `get_translatable_attributes` for example?

Reply:
This function is identical in all three resolvers, we could actually extract it outside the class and reuse it. Adding a short docstring would be then useful as well.

Reply:
Usually when dealing with single dataloader in resolvers, we compose func name like underscore + resolver name. For ex. `_resolve_attribute_values`. But in that case, yes, it should be extracted (and no underscore then).